### DISCUSSION TITLE
Swagger UI Landing page not appearing?

### DISCUSSION THREAD

**Discussion by RabeeHanna** [yahoo/elide#2829] [General]
I created a new project by following the start up guide step by step, as well as the swagger setup. When I run my Springboot application the default Spring security login page appears. I configured Spring security FilterChain to permit all HTTP requests and disabled CSRF too. Visiting the landing page after that shows the standard Whitelabel Page Error... 

I am able to access the /doc swagger document and it looks fine from there, but why is there no UI? In the Elide tutorial video (https://www.youtube.com/watch?v=WeFzseAKbzs) the person driving is able to run his app and navigate to the landing page (localhost:8080/) to see the UI. Am I missing something? Is this only for Heroku deployed apps? I am running my app locally.

**Comment by moizarafat** [comment]
We have a sample spring boot example repo which has all the configuration needed to avoid spring security login. https://github.com/yahoo/elide-spring-boot-example 

Please refer to that and let us know if it's still an issue.
